v_5_2_0/master_template.go

```diff
@@ -294,7 +294,6 @@ systemd:
         --kubeconfig=/etc/kubernetes/kubeconfig/kubelet.yaml \
         --node-labels="node.kubernetes.io/master,role=master,ip=${DEFAULT_IPV4},{{.Cluster.Kubernetes.Kubelet.Labels}}" \
         {{ if eq .Cluster.Kubernetes.CloudProvider "aws" }}
-        --cni-config-dir=/etc/cni/net.d \
```

Comment:
Just personal curiousity. Why are this not needed?

Reply:
this is the default settings, so there is no point of defining it again with the same value

Reply:
oh right, man. I did not look at the default parameters. thank you